docs/version0.35.md

```diff
@@ -0,0 +1,52 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# What's new in version 0.35.0
+
+The following new features and notable changes since version 0.33.1 are included in this release:
+
+- [New binaries and changes to supported environments](#binaries-and-supported-environments)
+- [Java dump file contains more information about waiting threads](#java-dump-file-contains-more-information-about-waiting-threads)
+
+## Features and changes
+
+### Binaries and supported environments
+
+OpenJ9 release 0.35.0 supports OpenJDK 8, 11, 17, and 18.
```

Comment:
We hope to include support for OpenJDK 19, but 0.35.0 will probably not include an update for 18.

Reply:
Have included this change in the issue (https://github.com/eclipse-openj9/openj9-docs/issues/976) that was created to add support for OpenJDK 19 in the related documents after it is confirmed.

Reply:
Independent of whether jdk19 is part of the 0.35.0 release, jdk18 will not be. I'm fine if you want to add 19 later once we're sure it will be in the 0.35.0 release, but the reference to 18 should be removed.

Reply:
I don't see jdk19 making the 0.35 release.